ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -120,8 +116,13 @@ private RequestMapper createRequestMapper() {
   }
 
   private Router router() {
```

Comment:
this really should be above the previous function (just for readability)

Reply:
done